src/plugins/mf_priority.cpp

```diff
@@ -45,6 +45,21 @@ extern "C" {
 // the association does not have permission to run jobs under
 #define INVALID_QUEUE -6
 
+// different codes to return as a result of looking up user/bank information:
+//
+// USER_NOT_FOUND: the user could not be found in the plugin map
+// BANK_SUCCESS: we found an entry for the passed-in user/bank
+// INVALID_BANK: the user specified a bank they don't belong to
+// NO_DEFAULT_BANK: the user does not have a default bank in the plugin map
+enum bank_info_codes {
+    USER_NOT_FOUND,
+    BANK_SUCCESS,
+    INVALID_BANK,
+    NO_DEFAULT_BANK
+};
```

Comment:
Suggestion: it is sometimes good practice to name enumerations with a common prefix, e.g. if it works here prefix all these values with `BANK_`: `BANK_USER_NOT_FOUND`, `BANK_NO_DEFAULT`, `BANK_INVALID` etc. This just helps when reading and writing code.

Also, just another suggestion, but is more typical to see the first enumeration value (i.e. `0`) be SUCCESS. Since that is more common practice, I'd suggest moving `BANK_SUCCES` to the first position of the enumeration.

Reply:
Thanks for these suggestions! I was unaware of the practices here (I think it's my first time using enumeration values? can't remember if i ever did this), so I'll make these changes to use a common `BANK` prefix as well as using the SUCCESS value as the first position. 👍